### DISCUSSION TITLE
Add Custom Field when fetching data

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6455] [Q&A] [answered]
```prisma
model User {
id
firstName
lastName
email
}
```

```ts
const result = await this.prisma.users({select: {id:true, email:true,  fullName:true }})  // fullName should equal firstName + lastName
```

final result
[
{
id: 1,
firstName: bishoy,
lastName: narouz,
fullName: bishoy narouz,
email: dd@dd.com
}
}

**Comment by ryands17** [comment, marked as answer]
@BishoyNarouz 👋 

This is currently not possible in the schema, but if you are using Postgres, you can use [generated fields](https://severalnines.com/database-blog/overview-generated-columns-postgresql) to have the same functionality.

I would suggest following these requests for more info and adding a 👍 to it as well so that we can look at the priority.
https://github.com/prisma/prisma/issues/3394
https://github.com/prisma/prisma/issues/2597

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
